bouncer.go

```diff
@@ -273,52 +275,61 @@ func handleNoStreamCache(bouncer *Bouncer, rw http.ResponseWriter, req *http.Req
 		return
 	}
 	if bouncer.crowdsecMode == liveMode {
-		setDecision(bouncer.cache, remoteHost, true, int64(duration.Seconds()))
+		setDecision(remoteHost, true, int64(duration.Seconds()))
 	}
 }
 
 func handleStreamCache(bouncer *Bouncer) {
 	// TODO clean properly on exit.
+	// Instead of blocking the goroutine interval for all the secondary node,
+	// if the master service is shut down, other goroutine can take the lead
+	// because updated routine information is in the cache
+	_, err := getDecision(cacheTimeoutKey)
+	if err == nil {
+		return
+	}
+	setDecision(cacheTimeoutKey, true, bouncer.updateInterval - 1)
```

Comment:
What is the purpose of bouncer.updateInterval - 1

So it blocks any other services that would try to update ?

What if the first step takes more than one seconde, then next run won't be started until next ticker